test/modules/aws-codedeploy-integration.ts

```diff
@@ -57,7 +57,7 @@ const getInstanceTypeOffering = async (availabilityZones: string[]) => {
       },
       {
         Name: 'instance-type',
-        Values: ['t2.micro', 't3.micro'],
+        Values: ['t*.medium', 't*.large'],
```

Comment:
`t*` works? Also, it should be `m*.large` for the second one?

Reply:
there can be also t*.large... but we could make it wider, and use t*.medium, t*.large, m*.large ?